include/libgit4cpp/Repository.h

```diff
@@ -233,8 +233,56 @@ class Repository
      * \param branch_name
     */
     bool branch_up_to_date(const std::string& branch_name);
+
 #endif
 
+
+    /**
+     * Create a new branch from the current branch.
+     * \param branch_name name of the new branch
+    */
```

Comment:
Mis-formatted throughout the whole PR:
```suggestion
     */
```

Reply:
I hope I found all of them...